app/client/components/accountoverview/product.tsx

```diff
@@ -75,94 +75,118 @@ export const Product = (props: ProductProps) => {
         margin-top: ${space[12]}px;
       `}
     >
-      <h2
+      <div
         css={css`
-          position: relative;
+          display: flex;
+          justify-content: space-between;
+          align-items: start;
           background-color: ${hasCancellationPending
             ? palette.neutral[97]
             : palette.brand[400]};
-          margin: 0;
-          padding: ${space[3]}px 12ch ${space[3]}px ${space[3]}px;
-          color: ${hasCancellationPending
-            ? palette.neutral[7]
-            : palette.neutral[100]};
-          ${titlepiece.small()};
-          font-weight: bold;
-          font-size: 17px;
           ${minWidth.tablet} {
```

Comment:
should this be...
```suggestion
          ${minWidth.mobileLandscape} {
```

Reply:
done in https://github.com/guardian/manage-frontend/pull/414/commits/ce6ba30b6fe5d5df12f3c637cbfa4f1056ee4f6f